chargedUp/src/main/java/frc/robot/Robot.java

```diff
@@ -51,7 +51,9 @@ public void robotPeriodic() {
 
   /** This function is called once each time the robot enters Disabled mode. */
   @Override
-  public void disabledInit() {}
+  public void disabledInit() {
+    m_robotContainer.disablePIDSubsystems();
```

Comment:
This disabled a function that doesn't have any functional code in it

Reply:
removed function 👍